platform/darwin/src/MGLMapSnapshotter.h

```diff
@@ -57,18 +63,26 @@ MGL_EXPORT
 
 @end
 
+#if TARGET_OS_IPHONE
 /**
  A block to processes the result or error of a snapshot request.
  
- The result will be either an `MGLImage` or a `NSError`
+ @param snapshot The `UIImage` that was generated or `nil` if an error occurred.
+ @param error The error that occured or `nil` when succesful.
+ */
+typedef void (^MGLMapSnapshotCompletionHandler)(UIImage* _Nullable snapshot, NSError* _Nullable error);
+#else
+/**
+ A block to processes the result or error of a snapshot request.
  
- @param snapshot The image that was generated or `nil` if an error occurred.
+ @param snapshot The `NSImage` that was generated or `nil` if an error occurred.
  @param error The eror that occured or `nil` when succesful.
  */
-typedef void (^MGLMapSnapshotCompletionHandler)(MGLImage* _Nullable snapshot, NSError* _Nullable error);
+typedef void (^MGLMapSnapshotCompletionHandler)(NSImage* _Nullable snapshot, NSError* _Nullable error);
+#endif
 
 /**
- A utility object for capturing map-based images.
+ An immutable utility object for capturing map-based images.
  */
```

Comment:
As tail work, it would be helpful to add a small code example of working with MGLMapSnapshotter.

Reply:
Tail work ticket in our ios-sdk repo https://github.com/mapbox/ios-sdk/issues/332

Reply:
I was referring to a short code example here inside the header, similar to what we do for classes like MGLShapeSource and MGLMapView, so that the example shows up at the beginning of the MGLMapSnapshotter class reference.